app/routes/endemics/number-of-fluid-oral-samples.js

```diff
@@ -0,0 +1,67 @@
+const Joi = require('joi')
+const session = require('../../session')
+const urlPrefix = require('../../config').urlPrefix
+const {
+  endemicsTestUrn,
+  endemicsNumberOfOralFluidSamples,
+  endemicsNumberOfOralFluidSamplesException,
+  endemicsTestResults
+} = require('../../config/routes')
+const {
+  endemicsClaim: { numberOfOralFluidSamples: numberOfOralFluidSamplesKey }
+} = require('../../session/keys')
+
+const pageUrl = `${urlPrefix}/${endemicsNumberOfOralFluidSamples}`
+
+module.exports = [
+  {
+    method: 'GET',
+    path: pageUrl,
+    options: {
+      handler: async (request, h) => {
+        const { numberOfOralFluidSamples } = session.getEndemicsClaim(request)
+        return h.view(endemicsNumberOfOralFluidSamples, {
+          numberOfOralFluidSamples,
+          backLink: `${urlPrefix}/${endemicsTestUrn}`
+        })
+      }
+    }
+  },
+  {
+    method: 'POST',
+    path: pageUrl,
+    options: {
+      validate: {
+        payload: Joi.object({
+          numberOfOralFluidSamples: Joi.string().pattern(/^\d+$/).max(6).required()
+            .messages({
+              'string.base': 'Enter the number of oral fluid samples collected',
+              'string.empty': 'Enter the number of oral fluid samples collected',
+              'string.max': 'The number of animals tested should not exceed 999999',
+              'string.pattern.base': 'Number of animals tested must only include numbers'
+            })
+        }),
+        failAction: async (request, h, error) => {
+          return h
+            .view(endemicsNumberOfOralFluidSamples, {
+              ...request.payload,
+              errorMessage: { text: error.details[0].message, href: '#numberOfOralFluidSamples' },
+              backLink: `${urlPrefix}/${endemicsTestUrn}`
+            })
+            .code(400)
+            .takeover()
+        }
+      },
+      handler: async (request, h) => {
+        const { numberOfOralFluidSamples } = request.payload
+        session.setEndemicsClaim(request, numberOfOralFluidSamplesKey, numberOfOralFluidSamples)
+
+        if (numberOfOralFluidSamples < 5) {
```

Comment:
we should avoid using magic numbers, can we please add the value `5` in a variable and give it descriptive name